contrib/drivers/drivers.go

```diff
@@ -57,6 +57,12 @@ type VolumeDriver interface {
 
 	DeleteSnapshot(opt *pb.DeleteVolumeSnapshotOpts) error
 
+	CreateVolumeGroup(opt *pb.CreateVolumeGroupOpts, group *model.VolumeGroupSpec) (*model.VolumeGroupSpec, int, error)
+
+	UpdateVolumeGroup(opt *pb.UpdateVolumeGroupOpts, group *model.VolumeGroupSpec, addVolumesRef []*model.VolumeSpec, removeVolumesRef []*model.VolumeSpec) (*model.VolumeGroupSpec, []*model.VolumeSpec, []*model.VolumeSpec, int, error)
```

Comment:
Same with last comment

Reply:
Can you explain why do you add so many parameters in volume group operation?

Reply:
When driver call UpdateVolumeGroup method, it may change the add volumes, remove volumes and group these three parameters or one of them or two of them. Different driver may take different strategies which refer to cinder.